### ISSUE TITLE
package.json and .eslintrc ignored

### ISSUE THREAD

**Issue by mercmobily** [ricardofbarros/linter-js-standard#233]
Hi,

Sorry if this is not the right repository for this report.
I installed the linter in atom as a dependency of linter-js-standard. I added this to my package.json:

````
  "eslintConfig": {
    "parserOptions": {
      "ecmaVersion": 6
    },
    "rules": {
      "no-var": "error"
    }
  }
````

However, no matter what I do, I just can't get any of the rules I set here to apply.
I also tried to create a `.eslintrc` file in my project root -- no luck.

Am I missing something?

**Comment by sonicdoe** [comment]
This Atom package only lints [JavaScript Standard Style](https://standardjs.com) but not any custom ESLint rules. If you’d like to change `standard`’s ESLint ruleset, your best option is probably to use [eslint-config-standard](https://github.com/standard/eslint-config-standard) and install [linter-eslint](https://github.com/AtomLinter/linter-eslint) for Atom. See also [I disagree with rule X, can you change it?](https://standardjs.com/index.html#i-disagree-with-rule-x-can-you-change-it)
